app/views/index_intl.scala.html

```diff
@@ -15,16 +15,15 @@ <h2 class="block__title">Subscribe to the Guardian <span class="break-tablet">di
             <div class="block__info">
                 <ul class="block__list">
                     <li class="block__list-item">14-day free trial</li>
-                    <li class="block__list-item"><strong>Plus, save 50% for the first 3 months</strong></li>
-                    @edition.digitalPackSaving.map {saving => <li class="block__list-item">Save up to @{saving}% on the app store price</li>}
+                    @edition.digitalPackSaving.map {saving => <li class="block__list-item"><strong>Save up to @{saving}% on the app store price</strong></li>}
                     <li class="block__list-item">Access to the Daily Edition, the digital version of the UK Guardian newspaper</li>
                     <li class="block__list-item">Access to the advert-free Guardian App Premium Tier</li>
                     <li class="block__list-item">Available on Apple, Android and Kindle Fire</li>
                     <li class="block__list-item">Use on up to 10 devices</li>
                 </ul>
             </div>
             <div class="block__footer">
-                <a class="button button--large button--primary button--arrow" href="@routes.PromoLandingPage.render(s"DBH80X", None).url?edition=@edition.id" data-test-id="subscriptions-@edition.id-digital">Subscribe now</a>
+                <a class="button button--large button--primary button--arrow" href="@routes.PromoLandingPage.render(s"DHOME${edition.id.toUpperCase}", None).url?edition=@edition.id" data-test-id="subscriptions-@edition.id-digital">Subscribe now</a>
```

Comment:
Is this meant to be "DHOME${edition.id.toUpperCase}**1**" or is the UK the only one ending in 1?